src/core/components/toast/styles.ts

```diff
@@ -0,0 +1,69 @@
+import {styled, keyframes, css} from 'styled-components'
+import {getTheme_v2} from '../../../theme'
+import {POPOVER_MOTION_CONTENT_OPACITY_PROPERTY} from '../../constants'
+import {Flex} from '../../primitives'
+import {ThemeProps} from '../../styles'
+
+export const TextBox = styled(Flex)`
+  overflow-x: auto;
+`
+
+const loadingAnimation = keyframes`
+  0% {
+    background-position: 100% 0;
+  }
+  100% {
+    background-position: 0 0;
+  } 
+`
+
+export function rootStyles(
+  props: {$duration?: number; tone: string} & ThemeProps,
```

Comment:
```suggestion
  props: {$duration?: number; tone: ThemeColorStateToneKey} & ThemeProps,
```

This should get around the type issues below!